MappaMundi/aStar/MMNode.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MMNode.swift
+//  MappaMundi
+//
+//  Created by Yang, Roy on 2019/6/7.
+//  Copyright © 2019 Mozilla Corporation. All rights reserved.
```

Comment:
Please use the mozilla licence header, thanks

Reply:
sure, updated.